packages/nx/src/hasher/hasher.ts

```diff
@@ -369,8 +369,8 @@ class TaskHasher {
     // if it's "run commands" we skip traversing since we have no info what this command depends on
     // for everything else we take the hash of the @nrwl package dependency tree
     if (
-      target.executor.startsWith(`@nrwl/`) &&
-      target.executor !== `@nrwl/workspace:run-commands`
```

Comment:
This always seemed wierd, do we no longer need it? I'd love a breakdown of what it was doing.